.fixtures.yml

```diff
@@ -3,9 +3,21 @@ fixtures:
     stdlib: 'https://github.com/puppetlabs/puppetlabs-stdlib'
     vcsrepo: 'https://github.com/puppetlabs/puppetlabs-vcsrepo'
     redis: 'https://github.com/arioch/puppet-redis'
+    python: 'https://github.com/voxpupuli/puppet-python'
+    epel:
+      repo: 'https://github.com/stahnma/puppet-module-epel.git'
     augeas_core:
       repo: https://github.com/puppetlabs/puppetlabs-augeas_core.git
       puppet_version: ">= 6.0.0"
     selinux_core:
       repo: https://github.com/puppetlabs/puppetlabs-selinux_core.git
       puppet_version: ">= 6.0.0"
+    python: 'https://github.com/stankevich/puppet-python'
```

Comment:
two python modules?

Reply:
Well spotted, I've removed the duplicated one.